tests/client_message_test.py

```diff
@@ -171,7 +171,7 @@ def test_set_flag_listener(self):
 
 class ClientMessageBuilderTest(unittest.TestCase):
     def test_message_accumulate(self):
-        message = client_authentication_codec.encode_request("user", "pass", "uuid", "owner-uuid", True, "PYH", 1)
+        message = client_authentication_codec.encode_request("user", "pass", "uuid", "owner-uuid", True, "PYH", 1, "3.10")
```

Comment:
use CLIENT_VERSION instead of 3.10? it will fail in the next release.

Reply:
It is just a test to message builder. None of these parameters are validated. How could it fail ?